### ISSUE TITLE
Communicate which timeline is being replaced upon failure in Tutorial 2 more clearly

### ISSUE THREAD

**Issue by kb173** [lizrad/recursio#179]
A tester was confused about the state of the timelines after failing (ghost dying) in Tutorial 2. He wasn't sure which information he's now replacing. Not sure how that can be fixed elegantly... maybe showing the ghost paths in that phase too would help?
